### PR TITLE
l10n: uk: v2.43 translation

### PR DESCRIPTION
**Pull request by arkid15r**

### CODE REVIEW

File: po/uk.po [hunk 1]
```diff
@@ -2969,9 +2964,8 @@ msgid "git bisect replay <logfile>"
 msgstr "git bisect replay <лог файл>"
 
 #: builtin/bisect.c:49
-#, fuzzy
 msgid "git bisect run <cmd> [<arg>...]"
-msgstr "git bisect run <команда>..."
+msgstr "git bisect run <команда> [<аргументи>...]"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "git bisect run <команда> [<аргумент>...]"
```

File: po/uk.po [hunk 2]
```diff
@@ -14119,18 +14094,17 @@ msgid "pack prefix to store a pack containing pruned objects"
...
 #: builtin/repack.c:1204
 msgid "cannot delete packs in a precious-objects repo"
 msgstr "неможливо видалити пакунки в precious-objects сховищі"
 
 #: builtin/repack.c:1342
-#, fuzzy, c-format
+#, c-format
 msgid "option '%s' can only be used along with '%s'"
-msgstr "'%s' не може використовуватись з '%s'"
+msgstr "опція \"%s\" можна використовувати тільки разом з \"%s\""
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "опцію \"%s\" можна використовувати тільки разом з \"%s\""
```

File: po/uk.po [hunk 3]
```diff
@@ -22090,29 +22056,29 @@ msgid "multi-pack-index hash version %u does not match version %u"
 msgstr "версія хешу multi-pack-index %u не збігається з версією %u"
 
 #: midx.c:175
-#, fuzzy
 msgid "multi-pack-index required pack-name chunk missing or corrupted"
-msgstr "multi-pack-index недостає необхідного фрагмента імені пакунка"
+msgstr ""
+"multi-pack-index необхідній шматок імені пакунка відсутній або пошкоджений"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
"multi-pack-index необхідний шматок імені пакунка відсутній або пошкоджений"
```

File: po/uk.po [hunk 4]
```diff
@@ -22090,29 +22056,29 @@ msgid "multi-pack-index hash version %u does not match version %u"
...
 
 #: midx.c:177
-#, fuzzy
 msgid "multi-pack-index required OID fanout chunk missing or corrupted"
-msgstr "multi-pack-index недостає необхідного OID fanout фрагмента"
+msgstr ""
+"multi-pack-index необхідній шматок розсіювання OID відсутній або пошкоджений"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
"multi-pack-index необхідний шматок розсіювання OID відсутній або пошкоджений"
```

File: po/uk.po [hunk 5]
```diff
@@ -22090,29 +22056,29 @@ msgid "multi-pack-index hash version %u does not match version %u"
...
 
 #: midx.c:179
-#, fuzzy
 msgid "multi-pack-index required OID lookup chunk missing or corrupted"
-msgstr "multi-pack-index недостає необхідного OID lookup фрагмента"
+msgstr ""
+"multi-pack-index необхідній шматок пошуку OID відсутній або пошкоджений"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
"multi-pack-index необхідний шматок пошуку OID відсутній або пошкоджений"
```

File: po/uk.po [hunk 6]
```diff
@@ -22090,29 +22056,29 @@ msgid "multi-pack-index hash version %u does not match version %u"
...
 
 #: midx.c:181
-#, fuzzy
 msgid "multi-pack-index required object offsets chunk missing or corrupted"
-msgstr "multi-pack-index недостає необхідного фрагмента зміщення обʼєктів"
+msgstr ""
+"multi-pack-index необхідній шматок зміщення обʼєктів відсутній або "
```
Comment (kasya) [inline po/uk.po]:
```suggestion
"multi-pack-index необхідний шматок зміщення обʼєктів відсутній або "
```

File: po/uk.po [hunk 7]
```diff
@@ -22241,7 +22206,7 @@ msgstr "Пошук файлів пакунків, на які є посилан
 msgid ""
 "oid fanout out of order: fanout[%d] = %<PRIx32> > %<PRIx32> = fanout[%d]"
 msgstr ""
-"невірна послідовность oid fanout: fanout[%d] = %<PRIx32> > %<PRIx32> = "
+"невірна послідовность oid розсіювання: fanout[%d] = %<PRIx32> > %<PRIx32> = "
```
Comment (kasya) [inline po/uk.po]:
```suggestion
"невірна послідовність oid розсіювання: fanout[%d] = %<PRIx32> > %<PRIx32> = "
```

File: po/uk.po [hunk 8]
```diff
@@ -18574,12 +18542,12 @@ msgid "commit date for commit %s in commit-graph is %<PRIuMAX> != %<PRIuMAX>"
...
 
 #: commit-graph.c:2825
-#, fuzzy, c-format
+#, c-format
 msgid ""
 "commit-graph has both zero and non-zero generations (e.g., commits '%s' and "
 "'%s')"
 msgstr ""
-"коміт-граф має ненульовий номер генерації для коміту %s, але нульовий деінде"
+"коміт-граф має як нульові, так і ненульові покоління (коміти \"%s\" і \"%s\")"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
"коміт-граф має як нульові, так і ненульові генерації (коміти \"%s\" і \"%s\")"
```

**Review by kasya — COMMENTED** [review]
👍🏼

**Review by arkid15r — COMMENTED** [review]
Great suggestions, thanks!

**Pull request merged**